Gemfile

```diff
@@ -154,12 +154,12 @@ end
 group :development do
   gem 'byebug', '~> 9.0.0' # 9.1 requires ruby 2.2
   gem 'debugger-linecache'
-  gem "newrelic_rpm", "~> 3.0"
+  #gem "newrelic_rpm", "~> 3.0"
```

Comment:
tbh, I would just remove it. I never really used it because I get what I need with mini-profiler.

Reply:
I ended up re-adding it. it's harmless.